src/backend/aspen/workflows/pangolin/save.py

```diff
@@ -71,7 +101,30 @@ def cli(pangolin_fh: io.TextIOBase, pangolin_last_updated: datetime):
             pathogen_genome.pangolin_probability = pango_info["probability"]  # type: ignore
             pathogen_genome.pangolin_version = pango_info["version"]  # type: ignore
             pathogen_genome.pangolin_output = pango_info["full_output"]  # type: ignore
-            session.commit()
+            current_chunk_size += 1
+
+            # Support populating the sample_lineages table.
+            if pathogen_genome.sample.lineages:
+                lineage = pathogen_genome.sample.lineages[0]
+                lineage.lineage = pango_info["lineage"]  # type: ignore
+                lineage.lineage_software_version = pango_info["version"]  # type: ignore
+                lineage.lineage_probability = pango_info["probability"]  # type: ignore
+                lineage.raw_lineage_output = pango_info["full_output"]  # type: ignore
+            else:
+                lineage = SampleLineage(
+                    sample=pathogen_genome.sample,
+                    lineage_type=lineage_type,
+                    lineage=pango_info["lineage"],  # type: ignore
+                    lineage_software_version=pango_info["version"],  # type: ignore
+                    lineage_probability=pango_info["probability"],  # type: ignore
+                    raw_lineage_output=pango_info["full_output"],  # type: ignore
+                )
+                pathogen_genome.sample.lineages.append(lineage)
+
+            if current_chunk_size > commit_chunk_size:
+                session.commit()
+                current_chunk_size = 0
```

Comment:
Commit every N rows instead of for every row. Should speed things up for large #'s of samples.